addons/info/src/index.js

```diff
@@ -34,20 +28,10 @@ const defaultMarksyConf = {
   ul: UL,
 };
 
-export function addInfo(storyFn, context, info, _options) {
```

Comment:
IMHO, we should never export a function to users if we don't need to, because then we are stuck supporting it unless we want to break the API.